docs/source/models.rst

```diff
@@ -1879,25 +1881,25 @@ module accept the following data formats as input, depending on the deployment f
 Commands
 ~~~~~~~~~
 
-* :py:func:`run-local <mlflow.sagemaker.run_local>` deploys the model locally in a Docker
-  container. The image and the environment should be identical to how the model would be run
-  remotely and it is therefore useful for testing the model prior to deployment.
+* :py:func:`run-local <mlflow.sagemaker.run_local>` deploys the model locally in a Docker container.
+  The image and the environment should be identical to how the model would be run remotely and it is
+  therefore useful for testing the model prior to deployment.
 
 * `build-and-push-container <cli.html#mlflow-sagemaker-build-and-push-container>`_ builds an MLfLow
   Docker image and uploads it to ECR. The caller must have the correct permissions set up. The image
   is built locally and requires Docker to be present on the machine that performs this step.
 
-* :py:func:`deploy <mlflow.sagemaker.deploy>` deploys the model on Amazon SageMaker. MLflow
-  uploads the Python Function model into S3 and starts an Amazon SageMaker endpoint serving
-  the model.
+* :py:func:`deploy <mlflow.sagemaker.SageMakerDeploymentClient>` deploys the model on Amazon
+  SageMaker. MLflow uploads the Python Function model into S3 and starts an Amazon SageMaker
+  endpoint serving the model.
 
 .. rubric:: Example workflow using the MLflow CLI
 
 .. code-block:: bash
 
     mlflow sagemaker build-and-push-container  - build the container (only needs to be called once)
-    mlflow sagemaker run-local -m <path-to-model>  - test the model locally
-    mlflow sagemaker deploy <parameters> - deploy the model remotely
+    mlflow deployments run-local -t sagemaker -m <path-to-model>  - test the model locally
+    mlflow deployments -t sagemaker create - deploy the model remotely
```

Comment:
```suggestion
    mlflow deployments run-local -t sagemaker -m <path-to-model> # test the model locally
    mlflow deployments -t sagemaker create # deploy the model remotely
```

Can we replace `-` with `#`? It looks like `- test the model locally` is a part of the command, but it's not.

Reply:
Absolutely! Fixed!